### PR TITLE
Update users-roles.md

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
updating the correct link

### Issues Resolved
No issue, just incorrect link on the page, duplicated from the example prior

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AntonEliatra** [comment]
@hdhalter just a tiny PR for incorrect link

**Review by hdhalter — APPROVED** [review]

File: _security/access-control/users-roles.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ See [YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#roles
 
 ### REST API
 
-See [Create role]({{site.url}}{{site.baseurl}}/security/access-control/api/#create-role).
+See [Patch role]({{site.url}}{{site.baseurl}}/security/access-control/api/#patch-role).
```
Comment (hdhalter) [inline _security/access-control/users-roles.md:103]: Should this be ({{site.url}}{{site.baseurl}}/security/access-control/api/#patch-role-mapping)?

Comment (Naarcha-AWS) [reply _security/access-control/users-roles.md:103]: That would be for editing the roles's mappings, not the role itself. In other words, the role needs to be created or edited first before a role can be mapped to a user.

**Review by Naarcha-AWS — APPROVED** [review]

File: _security/access-control/users-roles.md (hunk 1 above)
Comment (Naarcha-AWS) [reply _security/access-control/users-roles.md:103]: We also start talking about this on the line after this section, 105.

**Pull request merged**